### PR TITLE
Added 8 of my own projects

### PR DESCRIPTION
**Pull request by DenverCoder1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,16 +25,25 @@ Please add new repos in their programming language and keep it alphabetical. [Cl
 
 ## JavaScript
 - [Cloudflare Status Worker](https://github.com/WalshyDev/cf-status-worker) - Get a message in your Discord server when Cloudflare has an issue. Deployed with Cloudflare Workers
+- [Custom Icon Badges](https://github.com/DenverCoder1/custom-icon-badges) - Allows users to more easily use Octicons and their own icons and logos in shields.io badges
 - [Mayuko](https://github.com/mayukobot/mayuko-js) - General purpose Discord bot for all things anime, and some osu! functionality.
+- [Unicode Formatter](https://github.com/DenverCoder1/unicode-formatter) - Convert portions of text to fancy text using unicode fonts for use on Twitter and other sites that don't support rich text
```
Comment (TheLastProject) [inline README.md]: Please make sure there is at least one issue so people know what to do

File: README.md [hunk 2]
```diff
@@ -25,16 +25,25 @@ Please add new repos in their programming language and keep it alphabetical. [Cl
...
 
 ## Kotlin
 - [Android Password Store](https://github.com/Android-Password-Store/Android-Password-Store) - Android client for [pass](https://passwordstore.org)
 - [Valetudo Companion](https://github.com/Hypfer/valetudo-companion) -  Valetudo Companion App for Android
 
+## PHP
+- [Countdown](https://github.com/DenverCoder1/countdown) - Website to display a countdown timer to a specified date
+- [GitHub Readme Streak Stats](https://github.com/DenverCoder1/github-readme-streak-stats) - 🔥 Stay motivated and show off your contribution streak! 🌟 Display your total contributions, current streak, and longest streak on your GitHub profile README
+- [Readme Typing SVG](https://github.com/DenverCoder1/readme-typing-svg) - ⚡ Dynamically generated, customizable SVG that gives the appearance of typing and deleting text. Typing SVGs can be used as a bio on your Github profile readme or repository.
+- [Simple View Counter](https://github.com/DenverCoder1/Simple-View-Counter) - Simple view counter for websites and Github READMEs
```
Comment (TheLastProject) [inline README.md]: Please make sure there is at least one issue so people know what to do

File: README.md [hunk 3]
```diff
@@ -25,16 +25,25 @@ Please add new repos in their programming language and keep it alphabetical. [Cl
...
+
 ## Python
 - [F-Droid Server](https://gitlab.com/fdroid/fdroidserver/) - Tools for managing builds, indexes, updates, and deployments for F-Droid repositories.
 - [PhishBuster](https://github.com/VFXGamer/PhishBuster) - A Phishing Site Detection Tool (Web Based)
-- [Virtual-Assistant](https://github.com/Veershah26/virtual-assistant)- A Virtual Assistant, where you can add the feature you like as a module in the modules folder, the nameof the assistant would also be decided at the end of the Hacktoberfest with a vote between all the contributors.  
+- [table2ascii](https://github.com/DenverCoder1/table2ascii) - Package for converting 2D Python lists to fancy ASCII tables. Table2Ascii lets you display pretty tables in the terminal and on Discord.
```
Comment (TheLastProject) [inline README.md]: Please make sure there is at least one issue so people know what to do

**Review by TheLastProject — CHANGES_REQUESTED** [review]

**Comment by DenverCoder1** [comment]
I can't currently think of more issues that need solving so I have removed some of the projects for now.

**Pull request merged**